recipes-kernel/linux/linux-juno/0001-juno-base.dtsi-add-optee-firmware-entries.patch

```diff
@@ -0,0 +1,29 @@
+From e183b5827330b2cb5db12143ae7174fee7329f19 Mon Sep 17 00:00:00 2001
+From: Koen Kooi <koen.kooi@linaro.org>
+Date: Thu, 16 Nov 2017 12:16:56 +0100
+Subject: [PATCH] juno-base.dtsi: add optee firmware entries
+
+Signed-off-by: Koen Kooi <koen.kooi@linaro.org>
```

Comment:
upstream-status? along the kernel config changes, it's something we should upstream and request Arvind to merge.

Reply:
Done both